MapboxNavigation/RouteMapViewController.swift

```diff
@@ -258,6 +260,34 @@ extension RouteMapViewController: NavigationMapViewDelegate {
             return defaultReturn
         }
         
+        if let userLocation = mapView.userLocation, routeController.showCurrentWayNameLabel == true {
+            
+            // Somehow check the current style has mapbox streets
+            //
+            //let streets = MGLSource(identifier: "mapbox://mapbox-streets-v7")
+            //guard mapView.style!.sources.contains(streets) else {
+            //  return
+            //}
+            
+            let userPuck = mapView.view(for: userLocation)
+            if let userPuck = userPuck {
+                let features = mapView.visibleFeatures(in: userPuck.frame)
+                
+                for layer in features {
```

Comment:
`visibleFeatures(in:)` returns an array of features, not an array of layers. The iOS SDK currently lacks a way to tell what style layer each visible feature belongs to: mapbox/mapbox-gl-native#7162. As a workaround, you can call `visibleFeatures(in:styleLayerIdentifiers:)` multiple times, once per relevant layer.